migrations/20160912132241_create_risks_of_bias.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+exports.up = (knex) => {
+  const schema = knex.schema;
+
+  schema.createTable('risk_of_biases', (table) => {
+    table.uuid('id').primary();
+
+    table.uuid('trial_id')
+      .notNullable()
+      .references('trials.id');
+    table.text('source_id')
+      .notNullable()
+      .references('sources.id');
+    table.text('source_url')
+      .notNullable();
```

Comment:
The `study_id` isn't enough to uniquely identify a risk of bias, then? In that case, you should create a unique constraint for `source_url` and `study_id`.